examples/enclave_manager/Makefile

```diff
@@ -15,20 +15,19 @@
 # Set SGX_MODE default (which can be overridden with an environment variable):
 SGX_MODE?=SIM
 
-PY_VERSION=${shell python3 --version | sed 's/Python \(3\.[0-9]\).*/\1/'}
+PY_VERSION=${shell python3 --version | sed 's/Python \(3\.[0-9]\).*/\1/' | tr -d .}
```

Comment:
What is "tr -d" for. I don;t see this option in PY_VERSION in "tools/build/Makefile" below

Reply:
This is needed because in wheel package naming convention, python tag doesn't have dot.
for eg: cp36 for python version 3.6